l2-contracts/contracts/bridge/L2SharedBridge.sol

```diff
@@ -82,14 +82,13 @@ contract L2SharedBridge is IL2SharedBridge, Initializable {
         address _l1Token,
         uint256 _amount,
         bytes calldata _data
-    ) external payable override {
```

Comment:
Are you sure that it will work as the interface is `payable`

Reply:
I think it was because of the weth bridge. Now payable can be removed